### PR TITLE
Beta add tg process

### PR DESCRIPTION
**Pull request by time2die**

### CODE REVIEW

File: src/main/java/Main.java [hunk 1]
```diff
@@ -1,13 +1,57 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.telegram.telegrambots.ApiContextInitializer;
+import org.telegram.telegrambots.TelegramBotsApi;
+import org.telegram.telegrambots.api.methods.send.SendMessage;
+import org.telegram.telegrambots.api.objects.Update;
+import org.telegram.telegrambots.bots.DefaultBotOptions;
```
Comment (sputnikci) [inline src/main/java/Main.java]: \[Checkstyle\] ERROR: Unused import \- org\.telegram\.telegrambots\.bots\.DefaultBotOptions\.

File: src/main/java/Main.java [hunk 2]
```diff
@@ -1,13 +1,57 @@
...
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.exceptions.TelegramApiException;
+import org.telegram.telegrambots.exceptions.TelegramApiRequestException;
+import org.telegram.telegrambots.generics.BotOptions;
```
Comment (sputnikci) [inline src/main/java/Main.java]: \[Checkstyle\] ERROR: Unused import \- org\.telegram\.telegrambots\.generics\.BotOptions\.

File: src/main/java/Main.java [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
 
 /**
  * Created by time2die on 06.11.16.
  */
 @SpringBootApplication
+
 public class Main {
-    public static void main(String[] args) {
-        new SpringApplicationBuilder(Main.class).web(false).run(args);
+    private static final Logger logger = Logger.getLogger("Telegram Bots Api");
```
Comment (sputnikci) [inline src/main/java/Main.java]: \[Checkstyle\] ERROR: Name 'logger' must match pattern '^\(log|\[A\-Z\]\[A\-Z0\-9\]\*\(\_\[A\-Z0\-9\]\+\)\*\)$'\.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package org.time2java.tRussianBank.domain;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java]: \[Checkstyle\] ERROR: Unused import \- lombok\.NoArgsConstructor\.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * Created by time2die on 08.11.16.
+ */
+@Data
+@Builder
+@EqualsAndHashCode
+public class Status {
+    Integer bankCapital;
+    Integer debt;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java]: \[Checkstyle\] ERROR: Variable 'debt' must be private and have accessor methods\.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    Integer active;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java]: \[Checkstyle\] ERROR: Variable 'active' must be private and have accessor methods\.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+    Integer bankAge;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java]: \[Checkstyle\] ERROR: Variable 'bankAge' must be private and have accessor methods\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+import lombok.NoArgsConstructor;
+
+/**
+ * Created by time2die on 08.11.16.
+ */
+@Data
+@Builder
+@EqualsAndHashCode
+public class User {
+    String name ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'name' must be private and have accessor methods\.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    String vkID ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    String city ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'city' must be private and have accessor methods\.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+    Integer paymentNum ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'paymentNum' must be private and have accessor methods\.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+    Integer paymentSum ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'paymentSum' must be private and have accessor methods\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+    Integer debtCount ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+    Integer currentDeb ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'currentDeb' must be private and have accessor methods\.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+    Integer earlyReturn ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'earlyReturn' must be private and have accessor methods\.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+    Integer delayReturn ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'delayReturn' must be private and have accessor methods\.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: ';' is preceded with whitespace\.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+    Boolean hasLastMounthsPays ;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java]: \[Checkstyle\] ERROR: Variable 'hasLastMounthsPays' must be private and have accessor methods\.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:24]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/utils/ConvertTools.java [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.time2java.tRussianBank.domain.User;
+import org.time2java.tRussianBank.domain.gaAnswer;
+
+import java.util.List;
+
+/**
+ * Created by time2die on 17.11.16.
+ */
+public class ConvertTools {
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/utils/ConvertTools.java:11]: [Checkstyle] ERROR: Utility classes should not have a public or default constructor.

File: src/test/java/ValidateTest.java [hunk 19]
```diff
@@ -33,6 +37,48 @@ public void canReadResources() throws URISyntaxException, IOException {
         Assert.assertNotNull(getFileContent("users.json"));
     }
 
+
+    static private  gaAnswer getUsersAnswer(){
```
Comment (sputnikci) [inline src/test/java/ValidateTest.java:41]: [Checkstyle] ERROR: 'private' modifier out of order with the JLS suggestions.

Comment (sputnikci) [inline src/test/java/ValidateTest.java:41]: [Checkstyle] ERROR: '{' is not preceded with whitespace.

File: src/test/java/ValidateTest.java [hunk 20]
```diff
@@ -33,6 +37,48 @@ public void canReadResources() throws URISyntaxException, IOException {
...
+        ObjectMapper om = new ObjectMapper();
+        try {
+            return om.readValue(getFileContent("users.json"), gaAnswer.class);
+        } catch (IOException e) {
+            Assert.fail();
+        }
+        return null ;
```
Comment (sputnikci) [inline src/test/java/ValidateTest.java:48]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/test/java/ValidateTest.java [hunk 21]
```diff
@@ -33,6 +37,48 @@ public void canReadResources() throws URISyntaxException, IOException {
...
+    }
+
+    @Test
+    public void canParseUser(){
```
Comment (sputnikci) [inline src/test/java/ValidateTest.java:52]: [Checkstyle] ERROR: '{' is not preceded with whitespace.

File: src/main/java/Main.java [hunk 22]
```diff
@@ -1,13 +1,54 @@
...
```
Comment (sputnikci) [inline src/main/java/Main.java:18]: [Checkstyle] ERROR: Name 'logger' must match pattern '^(log|[A-Z][A-Z0-9]*(_[A-Z0-9]+)*)$'.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java (hunk 4 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java:6]: [Checkstyle] ERROR: Unused import - lombok.NoArgsConstructor.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+import lombok.NoArgsConstructor;
+
+/**
+ * Created by time2die on 08.11.16.
+ */
+@Data
+@Builder
+@EqualsAndHashCode
+public class Status {
+    Integer bankCapital;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java:15]: [Checkstyle] ERROR: Variable 'bankCapital' must be private and have accessor methods.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java (hunk 5 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java:16]: [Checkstyle] ERROR: Variable 'debt' must be private and have accessor methods.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java (hunk 6 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java:17]: [Checkstyle] ERROR: Variable 'active' must be private and have accessor methods.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java (hunk 7 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java:18]: [Checkstyle] ERROR: Variable 'bankAge' must be private and have accessor methods.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java [hunk 24]
```diff
@@ -0,0 +1,21 @@
...
+    Integer debt;
+    Integer active;
+    Integer bankAge;
+    Integer memberNum;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java:19]: [Checkstyle] ERROR: Variable 'memberNum' must be private and have accessor methods.

File: src/main/java/org/time2java/tRussianBank/domain/Status.java [hunk 25]
```diff
@@ -0,0 +1,21 @@
...
+    Integer city;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/Status.java:20]: [Checkstyle] ERROR: Variable 'city' must be private and have accessor methods.

File: src/main/java/org/time2java/tRussianBank/domain/User.java [hunk 26]
```diff
@@ -0,0 +1,25 @@
+package org.time2java.tRussianBank.domain;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
```
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:6]: [Checkstyle] ERROR: Unused import - lombok.NoArgsConstructor.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 8 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:15]: [Checkstyle] ERROR: Variable 'name' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:15]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 9 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:16]: [Checkstyle] ERROR: Variable 'vkID' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:16]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 10 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:17]: [Checkstyle] ERROR: Variable 'city' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:17]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 11 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:18]: [Checkstyle] ERROR: Variable 'paymentNum' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:18]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 12 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:19]: [Checkstyle] ERROR: Variable 'paymentSum' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:19]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 13 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:20]: [Checkstyle] ERROR: Variable 'debtCount' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:20]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 14 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:21]: [Checkstyle] ERROR: Variable 'currentDeb' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:21]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 15 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:22]: [Checkstyle] ERROR: Variable 'earlyReturn' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:22]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 16 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:23]: [Checkstyle] ERROR: Variable 'delayReturn' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:23]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/domain/User.java (hunk 17 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:24]: [Checkstyle] ERROR: Variable 'hasLastMounthsPays' must be private and have accessor methods.

Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/domain/User.java:24]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/main/java/org/time2java/tRussianBank/utils/ConvertTools.java (hunk 18 above)
Comment (sputnikci) [inline src/main/java/org/time2java/tRussianBank/utils/ConvertTools.java:11]: [Checkstyle] ERROR: Utility classes should not have a public or default constructor.

File: src/test/java/ValidateTest.java (hunk 19 above)
Comment (sputnikci) [inline src/test/java/ValidateTest.java:41]: [Checkstyle] ERROR: 'private' modifier out of order with the JLS suggestions.

Comment (sputnikci) [inline src/test/java/ValidateTest.java:41]: [Checkstyle] ERROR: '{' is not preceded with whitespace.

File: src/test/java/ValidateTest.java (hunk 20 above)
Comment (sputnikci) [inline src/test/java/ValidateTest.java:48]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/test/java/ValidateTest.java (hunk 21 above)
Comment (sputnikci) [inline src/test/java/ValidateTest.java:52]: [Checkstyle] ERROR: '{' is not preceded with whitespace.

File: src/test/java/ValidateTest.java [hunk 27]
```diff
@@ -33,6 +37,48 @@ public void canReadResources() throws URISyntaxException, IOException {
...
+        gaAnswer usersAnswer = getUsersAnswer() ;
```
Comment (sputnikci) [inline src/test/java/ValidateTest.java:53]: [Checkstyle] ERROR: ';' is preceded with whitespace.

File: src/test/java/ValidateTest.java [hunk 28]
```diff
@@ -33,6 +37,48 @@ public void canReadResources() throws URISyntaxException, IOException {
...
+        List<User> users = ConvertTools.convertAnswerToUsers(usersAnswer) ;
```
Comment (sputnikci) [inline src/test/java/ValidateTest.java:54]: [Checkstyle] ERROR: ';' is preceded with whitespace.

**Comment by time2die** [comment]
done

**Pull request merged**